packages/desktop-client/src/components/reports/graphs/net-worth-spreadsheet.js

```diff
@@ -13,18 +14,31 @@ import {
 import { AlignedText } from '../../common';
 import { index } from '../util';
 
-export default function createSpreadsheet(start, end, accounts) {
+export default function createSpreadsheet(
+  start,
+  end,
+  accounts,
+  conditions = [],
+) {
   return async (spreadsheet, setData) => {
     if (accounts.length === 0) {
       return null;
     }
 
+    let { filters } = await send('make-filters-from-conditions', {
+      conditions: conditions.filter(cond => !cond.customName),
+    });
+
     const data = await Promise.all(
       accounts.map(async acct => {
         let [starting, balances] = await Promise.all([
           runQuery(
             q('transactions')
-              .filter({ account: acct.id, date: { $lt: start + '-01' } })
+              .filter({
+                $and: filters,
```

Comment:
Does this still make sense as a “Net Worth” graph even if it is filtered?

Reply:
I asked ChatGPT, here are some of the things it came up with:

![image](https://user-images.githubusercontent.com/25517624/236522205-02ea75ca-a85b-4cf6-a92c-c1524ab6a524.png)